cmd/consensus/main.go

```diff
@@ -644,6 +592,100 @@ func main() {
 				return nil, fmt.Errorf("could not find latest finalized block and pending blocks: %w", err)
 			}
 
+			hotstuffModules = &consensus.HotstuffModules{
+				Forks:                nil,
+				Validator:            nil,
+				Notifier:             notifier,
+				Committee:            committee,
+				Signer:               signer,
+				Persist:              persist,
+				Aggregator:           nil,
+				QCCreatedDistributor: pubsub.NewQCCreatedDistributor(),
+			}
+
+			hotstuffModules, err = consensus.InitForks(
+				finalized,
+				node.Storage.Headers,
+				finalize,
+				hotstuffModules,
+				node.RootBlock.Header,
+				node.RootQC,
+			)
+			if err != nil {
+				return nil, err
+			}
+			hotstuffModules = consensus.InitValidator(mainMetrics, hotstuffModules)
```

Comment:
It's better to make assignment to hotstuffModules fields explicit, so that it's clear to see when hotstuffModule is initialized, there are 3 `nil` fields, and they are updated with 3 assignments. And after the 3 assignments, all modules are created (no more nil field).
```suggestion
			forks, err := consensus.InitForks(
				finalized,
				node.Storage.Headers,
				finalize,
				hotstuffModules,
				node.RootBlock.Header,
				node.RootQC,
			)
			if err != nil {
				return nil, fmt.Errorf("could not init Forks", err)
			}
			hotstuffModules.Forks = forks
			
			validators := consensus.InitValidator(mainMetrics, hotstuffModules)
			hotstuffModules.Validator = validator
```